source/extensions/common/wasm/wasm.cc

```diff
@@ -440,6 +440,7 @@ uint32_t httpCallHandler(void* raw_context, uint32_t uri_ptr, uint32_t uri_size,
 }
 
 uint32_t getTotalMemoryHandler(void*) { return 0x7FFFFFFF; }
+uint32_t _emscripten_get_heap_sizeHandler(void*) { return 0x7FFFFFFF; }
```

Comment:
Is this used anywhere?

Reply:
Yes it is registered and exported into the WASM VM.